CHANGELOG.md

```diff
@@ -38,6 +38,10 @@
  * Note that `email_verified == not email_verification_required`.
  * A data migration will be necessary.
 
+## v8.1.1
+
+* Bugfix: Don't show "passwords do not match" when the first password is invalid.
```

Comment:
I tried to find the fix...turns out that v8.1.0 is not an ancestor of `master`. 

![https://raw.githubusercontent.com/meshy/gifs/master/cersei-eye-roll.gif](https://raw.githubusercontent.com/meshy/gifs/master/cersei-eye-roll.gif)